### ISSUE TITLE
Destination is not provided, splitted css files would not be written

### ISSUE THREAD

**Issue by jesujcastillom** [VitaliyR/postcss-esplit#6]
I'm getting this error, with Webpack 4:

`(Emitted value instead of an instance of Error) postcss-esplit: Destination is not provided, splitted css files would not be written`

This is my `postcss.config.js`:

```javascript
module.exports = {
	plugins: {
		'autoprefixer': {},
		'postcss-esplit': {},
	},
};
```

Am I doing something wrong? Or is this plugin not supporting webpack?

**Comment by VitaliyR** [comment]
@jesujcastillom are you using extract-text plugin? 

Anyway, this plugin not support webpack.. yet. It tries to write the separated files in the same folder where source file located and `@import` it. You can disable write via `writeFiles: false`, but it will do nothing useful.

I will add support when I will have a time for that in future.

**Comment by jesujcastillom** [comment]
@VitaliyR no, I'm using [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin) reason being that extract-text is not supported in webpack 4, I'm curious about how does this plugin works, since it's the only "warning" (it doesn't emit any files) that I'm getting, and I saw that it comes from not having the to option, which is supposed to be set by webpack, according to [postcss](https://github.com/webpack-contrib/mini-css-extract-plugin). Will take a look if I get the time and check if I can help this..
